### PR TITLE
Fix RBAC UI

### PR DESCRIPTION
**Pull request by soupette**
### What does it do?

This PR adds a border radius on the Tab component in the RBAC view.

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/Roles/Tabs/Tab.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ const Tab = styled.div`
   flex: 1;
   padding: 1rem;
   text-align: center;
+  border-top-left-radius: ${({ isFirst, theme }) =>
```
Comment (mfrachet) [inline packages/strapi-admin/admin/src/components/Roles/Tabs/Tab.js:8]: I think you can do this without JavaScript using https://css-tricks.com/almanac/selectors/f/first-of-type/ and last-of-type

File: packages/strapi-plugin-i18n/admin/src/components/DropdownIndicator/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  justify-content: space-around;
+  background: #fafafb;
+  > svg {
+    align-self: center;
+    font-size: 11px;
+    color: #b3b5b9;
+  }
+`;
+
+const DropdownIndicator = ({ selectProps: { menuIsOpen } }) => {
```
Comment (mfrachet) [inline packages/strapi-plugin-i18n/admin/src/components/DropdownIndicator/index.js:19]: This one already exists in strapi-helper, I don't think we need to duplicate it

**Review by mfrachet — CHANGES_REQUESTED** [review]
I've added two suggestions/modifications to this one. But it works okay on my side 👍🏻

File: packages/strapi-plugin-i18n/admin/src/components/DropdownIndicator/index.js (hunk 2 above)
Comment (soupette) [reply packages/strapi-plugin-i18n/admin/src/components/DropdownIndicator/index.js:19]: Well it does not have the correct UI we need this one so its similar to the one used in the other react select.

File: packages/strapi-helper-plugin/lib/src/components/Select/index.js [hunk 3]
```diff
@@ -3,17 +3,18 @@ import ReactSelect, { components } from 'react-select';
...
-      <Carret fill={theme.main.colors.grey} />
-    </components.DropdownIndicator>
-  );
-};
+//   return (
+//     <components.DropdownIndicator {...props}>
+//       <Carret fill={theme.main.colors.grey} />
+//     </components.DropdownIndicator>
+//   );
+// };
```
Comment (HichamELBSI) [inline packages/strapi-helper-plugin/lib/src/components/Select/index.js:17]: Do you still need that comments?

File: packages/strapi-admin/admin/src/components/Roles/Tabs/Tab.js (hunk 1 above)
Comment (HichamELBSI) [reply packages/strapi-admin/admin/src/components/Roles/Tabs/Tab.js:8]: Yep, you can do it on the Flex component above Tab components 👍🏽

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/strapi-admin/admin/src/components/Roles/Tabs/TabsWrapper.js [hunk 4]
```diff
@@ -4,6 +4,14 @@ const TabsWrapper = styled.div`
   display: block;
   border-radius: ${({ theme }) => theme.main.sizes.borderRadius};
   box-shadow: ${({ theme }) => `0px 2px 4px 0px ${theme.main.colors.darkGrey}`};
+  .tabs {
```
Comment (mfrachet) [inline packages/strapi-admin/admin/src/components/Roles/Tabs/TabsWrapper.js:7]: I think you don't need to do it that way. It couples de Flex component to the tab. Instead, what you could do is to put the style in the `Tabs` component.

Something like:

```js
const TabsWrapper = styled.div`
  display: block;
  border-radius: ${({ theme }) => theme.main.sizes.borderRadius};
  box-shadow: ${({ theme }) => `0px 2px 4px 0px ${theme.main.colors.darkGrey}`};

  /* Here */
  ${Tab}:first-of-type {
     // do stuff
  }
`;
```

**Review by mfrachet — CHANGES_REQUESTED** [review]
Last request :p

**Review by mfrachet — APPROVED** [review]

**Comment by mfrachet** [comment]
fortawesome is not our buddy 😬 . but let's merge this and adjust later

**Pull request merged**